Resources/public/css/layout.css

```diff
@@ -83,11 +83,11 @@
 }
 
 .ez-notification-container {
-    position: absolute;
-    top: 0;
+    position: fixed;
+    bottom: 0;
     left: 0;
     width: 100%;
-    z-index: 100;
+    z-index: 999999;
```

Comment:
You should put 6000001 so that the notification bar is also on top of the SF toolbar with SF 2.7